src/core/Akka/Routing/Broadcast.cs

```diff
@@ -153,38 +153,28 @@ public override RouterConfig WithFallback(RouterConfig routerConfig)
 
         private RouterConfig OverrideUnsetConfig(RouterConfig other)
         {
-            if (other is NoRouter)
+            if (other is Pool pool)
```

Comment:
everything that isn't a ```Pool``` returned ```this``` , so this is the only real check we need. Same with the other routers.